systemtests/src/main/java/io/enmasse/systemtest/platform/Minikube.java

```diff
@@ -15,12 +15,16 @@
 import io.fabric8.kubernetes.client.ConfigBuilder;
 import io.fabric8.kubernetes.client.DefaultKubernetesClient;
 import io.fabric8.kubernetes.client.utils.HttpClientUtils;
+import io.fabric8.openshift.api.model.Route;
```

Comment:
why we need imports in this class? I don't see any use in code

Reply:
fixed